suites/squid/cephadm/tier0.yaml

```diff
@@ -0,0 +1,145 @@
+#===============================================================================================
```

Comment:
This test suite is part of smoke tests [1], can be removed from here.
[1] https://github.com/red-hat-storage/cephci/blob/master/suites/squid/smoke/bvt.yaml

Reply:
Done